lib/WP_Auth0_Api_Client.php

```diff
@@ -550,7 +550,7 @@ public static function create_client_grant( $app_token, $client_id ) {
 		$data = array(
 			'client_id' => $client_id,
 			'audience' => self::get_connect_info( 'audience' ),
-			'scope' => self::ConsentRequiredScopes()
+			'scope' => self::get_required_scopes()
```

Comment:
this add in the last PR?

Reply:
@cocojoe - I'm not sure what you're asking here.

Reply:
Point me to this method please

Reply:
@cocojoe - Couldn't find it? 😆

https://github.com/auth0/wp-auth0/blob/master/lib/WP_Auth0_Api_Client.php#L221